### PR TITLE
Fix 102 tod os voznyarskyy

### PR DESCRIPTION
**Pull request by VozNick**
Some TODOs are done, feel free to review and give feedback if needed )

### CODE REVIEW

**Review by vitaliimak — COMMENTED** [review]
It looks well!
Only one, could you run `mvn checkstyle:check` and fix errors?

**Comment by VozNick** [comment]
> It looks well!
> Only one, could you run `mvn checkstyle:check` and fix errors?
ok, I will

**Comment by VozNick** [comment]
ok, I will

File: server/src/main/java/com/lunchtime/controllers/FeedbackController.java [hunk 1]
```diff
@@ -1,33 +1,27 @@
...
+    private final FeedbackServiceImpl feedbackServiceImpl;
 
-    //TODO rename variable name to normal CamelCase
-    private final FeedbackServiceImplement feedbackServiceimplementation;
-
-    public FeedbackController(FeedbackServiceImplement feedbackServiceimplementation) {
-        this.feedbackServiceimplementation = feedbackServiceimplementation;
+    public FeedbackController(FeedbackServiceImpl feedbackServiceImpl) {
+        this.feedbackServiceImpl = feedbackServiceImpl;
     }
```
Comment (narekacy) [inline server/src/main/java/com/lunchtime/controllers/FeedbackController.java]: Due to SOLID (O = "Classes should be open for extension but closed for modification"), we should use `FeedbackService` here, not `FeedbackServiceImpl`.

Morover, we can use **[@RequiredArgsConstructor](https://projectlombok.org/features/constructor)** and save code with constructor.

File: server/src/main/java/com/lunchtime/models/Feedback.java [hunk 2]
```diff
@@ -45,17 +46,6 @@
...
-
-    public Feedback(String description, Boolean isActive, Date date,
-                    Person person, Restaurant restId, Integer counterLike, Integer counterDislike) {
-        this.description = description;
-        this.isActive = isActive;
-        this.date = date;
-        this.person = person;
-        this.restId = restId;
-        this.counterLike = counterLike;
-        this.counterDislike = counterDislike;
```
Comment (narekacy) [inline server/src/main/java/com/lunchtime/models/Feedback.java:60]: Can't undersand why all args constructor is removed?

Comment (narekacy) [reply server/src/main/java/com/lunchtime/models/Feedback.java:60]: Now I can, thanks :)

File: server/src/main/java/com/lunchtime/controllers/FeedbackController.java (hunk 1 above)
Comment (VozNick) [reply server/src/main/java/com/lunchtime/controllers/FeedbackController.java]: Yes, I agree with SOLID, thats why few days ago I mentioned about this in one of our meetings, and You said that you have already fixed this.
PS. ..my task is just fixing TODOs, not refactoring all project, 
PS. nonetheless I can fix it too)

**Review by Andrii35 — DISMISSED** [review]
Well done!

File: server/pom.xml [hunk 3]
```diff
@@ -102,7 +102,12 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
-	</dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
```
Comment (vitaliimak) [inline server/pom.xml:109]: There is another junit dependency above

**Review by vitaliimak — APPROVED** [review]

**Review by Andrii35 — APPROVED** [review]

**Comment by narekacy** [comment]
Server starts. I have entities in DB generated, but can't see them in browsers UI.
I don't know why but I have this in console

```
org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "alter table if exists event add column event_category_id int8 not null" via JDBC Statement
Caused by: org.postgresql.util.PSQLException: ERROR: column "event_category_id" contains null values

org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "alter table if exists feedback add column restaurant_id int8 not null" via JDBC Statement
Caused by: org.postgresql.util.PSQLException: ERROR: column "restaurant_id" contains null values

org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "alter table if exists event add constraint FKnk9w760hnelms32e9oqecw6ru foreign key (event_category_id) references event_category" via JDBC Statement
Caused by: org.postgresql.util.PSQLException: ERROR: column "event_category_id" referenced in foreign key constraint does not exist

org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL "alter table if exists feedback add constraint FKt9gnevqw4oo5jnbv4e68tu0ua foreign key (restaurant_id) references restaurant" via JDBC Statement
Caused by: org.postgresql.util.PSQLException: ERROR: column "restaurant_id" referenced in foreign key constraint does not exist
```

**Review by Melnyk9506 — APPROVED** [review]
looks well

**Comment by vitaliimak** [comment]
@narekacy you should clear database

**Comment by narekacy** [comment]
@vitalii-mak tried it already. Doesn't help

**Comment by narekacy** [comment]
@vitalii-mak and now works

**Comment by narekacy** [comment]
@vitalii-mak I don't know why but when I do what is needed, it doesn't works. And then you tell the same thing to do, and it works. Some mystic, Vitalik)

**Pull request merged**